config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/cephcsi/ceph-csi-operator
+  newTag: latest
```

Comment:
Due to which step this got changed? am looking at to see how to make of this in upstream itself.

Reply:
`cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)` This command generates it. It will be done in upstream as well once someone change the image. As this command is present there as well.

Reply:
i just did `make build-installer ` as the image is changed we need this in upstream as well. Can you please push this change to the upstream branch

Reply:
The change is pushed